openmetadata-service/src/main/java/org/openmetadata/service/resources/services/messaging/MessagingServiceResource.java

```diff
@@ -345,6 +346,26 @@ public Response delete(
     return delete(uriInfo, securityContext, id, recursive, hardDelete);
   }
 
+  @PUT
+  @Path("/restore")
```

Comment:
Can we have path /restore/{id}?

Reply:
discussed offline:
PUT/POST requires a body and Jersey client throws an exception when there is no body